svn_extractor.py

```diff
@@ -8,13 +8,18 @@
 import traceback
 import re
 
+def getext(filename):
+    name, ext = os.path.splitext(filename)
+    return ext[1:]
```

Comment:
Actually in your example os.path.splitext('XYZ.exe.jpg') returns ('XYZ.exe', '.jpg') so ext becomes '.jpg'. I used [1:] to get rid of '.' at the begining.